docs/KernelFeatures/UsingSecureVault.md

```diff
@@ -106,3 +106,47 @@ All the Secret Repository implementations should derive from the Secret Reposito
             return new byte[0];
         }
     }
+```
+
+## Accessing Secure Vault in non OSGI environment.
+
+SecureVault functionality can access in non OSGI mode by following the below steps.
+
+1. Add secure vault dependency to the client project.
+
+        <dependency>
+            <groupId>org.wso2.carbon</groupId>
+            <artifactId>org.wso2.carbon.securevault</artifactId>
+            <version>${carbon.kernel.version}</version>
+        </dependency>
+
+2. Following Client class will initialize the secure vault to invoke resolve function to get the secret.
+```java
+import org.wso2.carbon.securevault.SecureVaultInitializer;
+import org.wso2.carbon.securevault.exception.SecureVaultException;
+
+/**
+ * Responsible for initialize the secure vault and invoke secure vault functionality
+ *
+ */
+public class SecureVaultClient
+{
```

Comment:
Is this formatting correct? Don't we need to end the line with the { even though this is in a  readme?